lib/modules/auth/signin.dart

```diff
@@ -0,0 +1,209 @@
+                        textInputAction: TextInputAction.done,
+                        keyboardType: TextInputType.visiblePassword,
+                        suffixIcon: isObscure
+                            ? Icons.remove_red_eye_outlined
+                            : Icons.remove_red_eye,
+                        obscureText: isObscure,
+                        suffixIconPress: () {
+                          setState(() {
+                            isObscure = !isObscure;
+                          });
+                        },
+                      ),
+                      SizedBox(
+                          height: MediaQuery.of(context).size.height * 0.08),
+                      Padding(
+                        padding: EdgeInsets.symmetric(
+                            horizontal:
+                                MediaQuery.of(context).size.width * 0.05),
+                        child: CustomButton(
+                          text: AppStrings.signIn,
+                          onPressed: () {
+                            if (signinFormKey.currentState!.validate()) {
+                              ScaffoldMessenger.of(context).showSnackBar(
+                                const SnackBar(
+                                  content: Text('Hey! There '),
+                                  backgroundColor: AppColors.inputLabelColor,
+                                ),
+                              );
+                            }
+                          },
+                        ),
+                      ),
+                      SizedBox(
+                          height: MediaQuery.of(context).size.height * 0.05),
+                      Padding(
+                        padding: EdgeInsets.symmetric(
+                            horizontal:
+                                MediaQuery.of(context).size.width * 0.05),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            Row(
+                              children: <Widget>[
+                                Checkbox(
+                                  shape: RoundedRectangleBorder(
+                                    borderRadius: BorderRadius.circular(2.0),
+                                  ),
+                                  side: MaterialStateBorderSide.resolveWith(
+                                    (states) => const BorderSide(
+                                        width: 1.0,
+                                        color: AppColors.lightYellow),
+                                  ),
+                                  value: rememberMe,
+                                  activeColor: Colors.transparent,
+                                  onChanged: (bool? value) {
+                                    setState(() {
+                                      rememberMe = value!;
```

Comment:
Please avoid force unwrap without checking for null value.

Reply:
Resolved